docs/data/material/components/dialogs/dialogs.md

```diff
@@ -168,8 +168,39 @@ The package [`material-ui-confirm`](https://github.com/jonatanklosko/material-ui
 
 Follow the [Modal accessibility section](/material-ui/react-modal/#accessibility).
 
-## Experimental APIs
-
-### Imperative API
-
-You can create and manipulate dialogs imperatively with the [`useDialog`](https://mui.com/toolpad/core/react-use-dialogs/) API in `@toolpad/core`. This API provides state management for opening and closing dialogs and for passing data to the dialog and back. It allows for stacking multiple dialogs. It also provides themed alternatives for `window.alert`, `window.confirm` and `window.prompt`.
+## Toolpad
+
+### `useDialogs`
```

Comment:
To match with https://mui.com/toolpad/core/react-use-dialogs/. code in headers almost never seem appropriate (unless you are in API reference). A rule Sam wanted to enforce I believe

```suggestion
### useDialogs
```